ts/Core/Tooltip.ts

```diff
@@ -558,6 +558,53 @@ class Tooltip {
                 .attr({ zIndex: 8 })
                 .shadow(options.shadow ?? !options.fixed)
                 .add();
+
+            // #23303
+            if (this.shouldStickOnContact() && this.label.element) {
+                const tooltipElement = this.label.element;
+
+                U.clearTimeout(this.hideTimer);
+
+                addEvent(tooltipElement, 'mouseover', (): void => {
+                    U.clearTimeout(this.hideTimer);
+                });
+
+                addEvent(
+                    tooltipElement,
+                    'mouseout',
+                    (e: PointerEvent): void => {
+                        const relatedTarget = e.relatedTarget as any;
```

Comment:
There should be something better than `as any`. Could there be something better than `as DOMElementType|null` without any bigger changes?